client/src/components/participants/index.tsx

```diff
@@ -1,57 +1,73 @@
 'use client';
 
-import { useRef, useState } from 'react';
+import { useState } from 'react';
 import { useTranslations } from 'use-intl';
 
-import { useOutsideClick } from '@/hooks';
 import { IUsersNormalizer } from '@/types';
 
-import { UserIcon } from './user-icon';
-import { DropdownList } from './dropdown';
+import { UserIcon } from '../user-icon';
+import { DropdownItem, DropdownList } from '../dropdown';
 
 import { getStyles } from './style';
 
 interface IParticipantsProps {
-  small?: boolean;
+  width?: number;
+  isTask?: boolean;
   users: IUsersNormalizer[];
+  dropdownClassName?: string;
 }
 
-export const Participants = ({ users, small }: IParticipantsProps) => {
-  const ref = useRef<HTMLDivElement>(null);
+const maxUser = 5;
```

Comment:
I'd rename it by adding some extra adjectives to explain the variable better.
Is that board max users or task max users?

Kindly ask you to rename it.